packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -615,7 +629,7 @@ export class PersonalDetailsForm extends React.Component {
                                             {/*        /> */}
                                             {/*    )} */}
                                             {/* </fieldset> */}
-                                            {this.props.is_eu && (
+                                            {this.props.is_mf && (
```

Comment:
How about `Tax information` for IOM clients?

Reply:
This should not be shown for IOM clients as they do not need to set it upon registration